app/controllers/start_group_controller.rb

```diff
@@ -2,7 +2,9 @@ class StartGroupController < ApplicationController
   def new
     @errors = []
     @group = Group.new
-    if current_user.present? && ENV['NEW_GROUPS_ON_ANGULAR']
```

Comment:
Hm, this feels complicated to me. What if we dropped the NEW_GROUPS_ON_ANGULAR setting (I don't think we need it), and used the `current_user_or_visitor` helper, which would make it so we don't have to make changes when we hop to logged out angular?

```
if current_user_or_visitor.angular_ui_enabled?
  render :enable_angular
else
  render :new
end
```

?